Makefile

```diff
@@ -77,3 +77,4 @@ env-down:                       ## Stop development environment.
 ci-reviewdog:                   ## Runs reviewdog checks.
 	$(BIN_PATH)/golangci-lint run -c=.golangci-required.yml --out-format=line-number | $(BIN_PATH)/reviewdog -f=golangci-lint -level=error -reporter=github-pr-check
 	$(BIN_PATH)/golangci-lint run -c=.golangci.yml --out-format=line-number | $(BIN_PATH)/reviewdog -f=golangci-lint -level=error -reporter=github-pr-review
+	$(BIN_PATH)/go-consistent -pedantic -exclude "tests" ./... | $(BIN_PATH)/reviewdog -f=go-consistent -name='Required go-consistent checks' -reporter=github-pr-check
```

Comment:
consider also add  the `go-consistent ` step to the `check-style` target, to let devs run the same checks locally

Reply:
Right, it was already added here https://github.com/percona/pmm-admin/blob/54dca08a7cba9adc61008cc06530f88e5cf88315/Makefile#L61